xml/preinstall-prepare-deployer.xml

```diff
@@ -148,6 +148,17 @@
    Guide</citetitle>.
   </para>
 
+
+  <important>
+   <title>Disable Default Firewall</title>
+   <para>
+    The default firewall must be disabled, as &productname; enables it's own
+    firewall during deployment. You can disable the default firewall in the "Installation
+    Settings" screen of the installation wizard.
```

Comment:
If you want to use quotes, use `<quote/>`, otherwise `<guimenu/>` is the most suitable tag.